packages/cohort_sdk_js/src/installer/mod.rs

```diff
@@ -0,0 +1,173 @@
+use std::fmt::Display;
```

Comment:
This package is draft from early Anil work. I will be moving it into its own "home" when building JS Replicator.